app/lib/BUILD

```diff
@@ -0,0 +1,14 @@
+load("@npm_bazel_typescript//:index.bzl", "ts_library")
+
+package(default_visibility = ["//visibility:public"])
+
+ts_library(
+    name = "lib",
```

Comment:
let's put all of this flame chart related stuff into it's own package that's like /app/flame_chart

i feel like the whole /lib/components thing just makes it harder to decide where things belong.

we can always break out parts that are re-used if that ever ends up happening

Reply:
I agree that this horizontal scrollbar probably won't be used outside of a chart scenario. In the future we might want to consider adding a `lib/components` directory for stuff that gets reused in lots of places.

WDYT about `//app/invocation/flame_chart` since I'm not sure a flame chart will make sense anywhere else?